### PR TITLE
Eventshifts

### PR DESCRIPTION
**Pull request by shwetamurali**
## PR Title/Tagline

Add event shifts in order to create a new object separate from the event

### How to Test

1. Please add or build upon a testing card to the [Notion page](http://notion.so) and link it here

### Important Changes

- Database change
- Changed frontend form input

### Related Github Issues

- #7 

### Related PRs

- #PRNUMBER

### Checklist

- [ ]  Database schema docs have been updated or are not necessary
- [ ]  Code follows design and style guidelines
- [ ]  Code is commented with doc blocks
- [ ]  Tests have been written and executed or are not necessary
- [ ]  Latest code has been rebased from base branch (usually `develop`)
- [ ]  Commits follow guidelines (concise, squashed, etc)
- [ ]  Github issues have been linked in relevant commits
- [ ]  Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### CODE REVIEW

File: backend/src/models/event.js [hunk 1]
```diff
@@ -31,14 +31,18 @@ const eventSchema = new mongoose.Schema(
...
-    max_volunteers: {
-      type: Number,
-      required: true
-    },
     external_links: {
       type: [String],
       default: [],
       required: false
+    },
+    max_shifts: {
```
Comment (azeng8100) [inline backend/src/models/event.js]: What is max shifts for?

**Review by azeng8100 — CHANGES_REQUESTED** [review]
Steps to recreate a bug:

1. Create two shifts and fill them in w/ random values
2. Create another shift that's empty
3. Delete the 2nd shift

Expected behavior: 2nd filled in shift is deleted. Left with 1st filled in shift and an empty shift
Bug: 2nd filled in shift is not deleted, empty shift is gone

Also can be replicated with deleting the first shift

File: backend/src/models/event.js (hunk 1 above)
Comment (vpatakottu) [reply backend/src/models/event.js]: I just realized I forgot to push the changes in the backend folder 😅. That should be deleted now!

**Comment by azeng8100** [comment]
Bug 2: Cannot delete multiple events
I created 3 events and deleted 1 event. I was unable to delete another event since the "Delete" button is faded/unclickable
image

Bug 3: Cannot see edit after submitting
Initial view:
image
Properly updated in database:
image
Cannot see new email after hitting edit again
image

Missing feature: Cannot edit shifts
1. We can remove max_shifts from here
2. Cannot view existing shifts and change their time
image

**Review by azeng8100 — APPROVED** [review]

**Pull request merged**